ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -17,7 +17,7 @@ public extension Colors {
     }
 
     internal struct NavigationBar {
-        static var background = UIColor(light: surfacePrimary, dark: gray900)
+        static var background: UIColor = surfacePrimary
```

Comment:
this has bigger consequences on all of our navigation bar colors. especially on dark mode split mode cases with dark elevated concepts. should it be part of this change? shouldn't cab has its own background color?